src/main/java/com/lambdaschool/tiemendo/model/Transaction.java

```diff
@@ -3,9 +3,11 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
 import javax.persistence.*;
+import java.lang.reflect.Array;
```

Comment:
whats up with this Reflection array?

Reply:
That is just miscellaneous and left over.  I was going through different solutions for totaling TransactionItems.  Didn't know you were working on it as well.  Forgot to delete after I resolved merge conflicts.  Deleted and pushed again.